<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transcoder Mesa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelEstadoDeTranscodificacion">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>Estado de Transcodificaci√≥n: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelDestino">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="text">
          <string>Destino:</string>
         </property>
         <property name="buddy">
          <cstring>textBoxRutaDeDestino</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="textBoxRutaDeDestino">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>/home/madmin/cat.mxf</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonDeSeleccionDeDestino">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelOrigen">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
         <property name="text">
          <string>Origen :</string>
         </property>
         <property name="buddy">
          <cstring>textBoxRutaDeOrigen</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textBoxRutaDeOrigen">
         <property name="text">
          <string>/home/madmin/cat.mp4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="botonDeSeleccionDeOrigen">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1" rowspan="2" colspan="2">
    <widget class="QPushButton" name="botonDeComenzar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Transcodificar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="indentation">
      <number>2</number>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Material</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="text">
       <string>unPrimerMaterialCualquiera.mp4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ERROR</string>
      </property>
      <property name="text">
       <string>unSegundoMaterialCualquiera.mp4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>YA EXISTE</string>
      </property>
      <property name="text">
       <string>unTercerMaterialCualquiera.mp4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PROCESANDO</string>
      </property>
      <property name="text">
       <string>unCuartoMaterialCualquiera</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
